cgi-bin/DW/Console/Command/ResetPBEMToken.pm

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/perl
+#
+# DW::Console::Command::ResetPBEMToken
+#
+# Console command for resetting PBEM tokens
+#
+# Authors:
+#      Adam Bernard <user name="pseudomonas" site="dreamwidth.org"">
```

Comment:
Don't use the user tag syntax here. The URL syntax you used in your previous PR is fine, though!